include/js/Inventory.js

```diff
@@ -1976,7 +1976,10 @@ function handleProductAutocompleteSelect(obj) {
 	document.getElementById('comment'+no).innerHTML = obj.result.meta.comments;
 	var currency = document.getElementById('inventory_currency').value;
 	if (obj.result.pricing.multicurrency[currency] != undefined && gVTModule != 'PurchaseOrder' && gVTModule != 'Receiptcards') {
-		document.getElementById('listPrice'+no).value = obj.result.pricing.multicurrency[currency].converted_price;
+		if (Object.keys(obj.result.pricing.multicurrency).length == 1 && obj.result.pricing.multicurrency[currency].actual_price != obj.result.pricing.unit_price) {
```

Comment:
is this check correct? shouldn't the actual price be different from the unit price for all currencies except the base currency?

If the base currency is EUR the unit price and the actual price in EUR should be the same, but the actual price in the other currencies may be different.  Am I wrong or is the "currency" variable always the base currency not the selected currency in the record?

Reply:
Well I didn't write the MC code but yes, the `currency` variable is the currently selected currency, I just doublechecked that. 

The assumption I do is this: since we basically decided that **every** product has an entry in `productcrurrencyrel` for every currency it is active for and **every** product should at least have one currency (and that should be the base currency if it's only one), we can state that there will **always** be at least one entry in the `multicurrency` object that is returned from the database. I just checked that in the demo: you can't disable the base currency, it'll just re-enable that during the save.

That means that _when_ there is only one entry in that object (first part of the check), that should be equal to the `unit_price` (second part of the check).

Please verify and doublecheck this and let me know if you have questions